sampleSpecs/reservednames.smithy

```diff
@@ -146,3 +151,17 @@ structure Scala3ReservedKeywords {
     export: String
     enum: String
 }
+
+@smithy4s.meta#adt
+@trait
```

Comment:
**thought:** maybe we should just make this illegal? using trait and adt like that

Reply:
I don't think it should be illegal, the semantics of these two being completely orthogonal. I do however think that dynamic bindings would solve it.